.travis.yml

```diff
@@ -5,7 +5,7 @@ jdk:
 stages:
   - name: test
   - name: release
-    if: (branch = master) AND (type = push OR tag IS present)
+    if: (branch = master) AND (type = push) OR (tag IS present)
```

Comment:
What's the precedence here?

```
((branch = master) AND (type = push)) OR (tag IS present)
(branch = master) AND ((type = push) OR (tag IS present))
```

Reply:
years of computer science tell me `AND` associates stronger than `OR`, but I won't take my chances and I'll add the parens :D

Reply:
I've written two Scala code formatters handling infix operator precedence and I can never remember how it goes 😅

Reply:
It's super easy in Scala: you just need to remember a fixed list of 500 symbols and their precedences 😝